src/read.c

```diff
@@ -170,31 +172,31 @@ static void ReadFuncCallOption(ReaderState * rs, TypSymbolSet follow)
     if (rs->s.Symbol == S_IDENT) {
         rnam = RNamName(rs->s.Value);
         Match(&rs->s, S_IDENT, "identifier", S_COMMA | follow);
-        TRY_IF_NO_ERROR { IntrFuncCallOptionsBeginElmName(rnam); }
+        TRY_IF_NO_ERROR { IntrFuncCallOptionsBeginElmName(&rs->intr, rnam); }
```

Comment:
Most of the changes below are about adding `&rs->intr` as first argument to `IntrFoobar` calls.